src/styles/_variables.scss

```diff
@@ -11,15 +11,14 @@ $spacing-8: 48px;
 $spacing-9: 64px;
 
 // Font sizes
-$font-size-10: 10px;
-$font-size-12: 12px;
-$font-size-14: 14px;
-$font-size-16: 16px;
-$font-size-18: 18px;
-$font-size-20: 20px;
-$font-size-24: 24px;
-$font-size-32: 32px;
-$font-size-40: 40px;
+$font-size-1: 2.71rem;
```

Comment:
Not a big fan of having random decimal values here. Would always prefer the calculation as part of some mixin. so here we could just have `$font-size-10 : pxToRem(10)` or even use pxToRem(10) directly where we are using it.